.github/workflows/dev-workflow.yml

```
@@ -53,27 +53,80 @@ jobs:
-          FIREBASE_STORAGE_BUCKET: "${{ secrets.FIREBASE_STORAGE_BUCKET }}"
-          FIREBASE_MESSAGING_SENDER_ID: "${{ secrets.FIREBASE_MESSAGING_SENDER_ID }}"
-          FIREBASE_APP_ID: "${{ secrets.FIREBASE_APP_ID }}"
-          FIREBASE_MEASUREMENT_ID: "${{ secrets.FIREBASE_MEASUREMENT_ID }}"
 
           API_ENDPOINT: https://api.dev.dagit.club
         run: |
           npm i -g aws-cdk
           make deploy-web
 
+main-ui:
+    runs-on: ubuntu-22.04
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)